python/python/opm/io/deck/__init__.py

```diff
@@ -1 +1,32 @@
 from opm._common import DeckKeyword
+from opm._common import DeckItem
+
+
+def defaulted_deckitem(self):
+    return self.__defaulted(0)
```

Comment:
Hardcoding the `0` means that the code will not work for the "data" items which have size greater than 1 - e.g. `PORO` or the tables; that is OK I guess - you are probing a peculiarity/design mistake in the original C++ api  - but maybe just add a comment here?

Reply:
I see that get_SI_data_list() and get_raw_data_list() can be used to get/extract data from arrays like PORO and PERMX, SWOF and so on.  

```python
@property
def get_data_deckkeyword(self):
    return self[0][0].get_raw_data_list()

setattr(DeckKeyword, "data", get_data_deckkeyword)
```

this can then be used like this

```python
permx =  dkw_permx.data
self.assertEqual(len(permx), 300)
```

Reply:
I'm not sure I understand the issue here and which comment is needed. May be we can pick up the phone on Monday